mautic/mautic
# Unpublished custom fields still visible in segments fix

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N 
| Automated tests included? | N
| Related user documentation PR URL |  N
| Related developer documentation PR URL | N 
| Issues addressed (#s or URLs) | [#5801](https://github.com/mautic/mautic/issues/5801)
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Fixes segments filter by removing custom field that should be hidden.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Unpublish any custom field
2. Setup segment filter
3. Custom field will still be there (same if field would be published but disabled for segments)

#### Steps to test this PR:
1. Checkout branch and test again steps to reproduce the bug

Both unpublish and disable for segments separately should hide custom field from filter form
